plugin/trino-exasol/src/test/java/io/trino/plugin/exasol/TestExasolConnectorSmokeTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.exasol;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.jdbc.BaseJdbcConnectorSmokeTest;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+
+import static io.trino.plugin.exasol.ExasolQueryRunner.createExasolQueryRunner;
+
+public class TestExasolConnectorSmokeTest
+        extends BaseJdbcConnectorSmokeTest
```

Comment:
BaseConnectorTest covers the test coverages of BaseJdbcConnectorSmokeTest. Running these tests with the same doesn't make sense. Let's remove this class.